src/script/api/Doxyfile_GS.in

```diff
@@ -10,8 +10,8 @@
 #---------------------------------------------------------------------------
 DOXYFILE_ENCODING      = UTF-8
 PROJECT_NAME           = "OpenTTD GameScript API"
-PROJECT_NUMBER         = $(VERSION)
-OUTPUT_DIRECTORY       = ../../../docs/gamedocs/
+PROJECT_NUMBER         = ${REV_VERSION}
+OUTPUT_DIRECTORY       = ${CPACK_BINARY_DIR}/docs/gamedocs/
```

Comment:
`gamedocs` always has been an odd name, as it is wrong in so many ways. Possibly a good moment to rename it?

`gs-api`? As that is what it is called here?
Maybe also change AI to be `ai-api` too while at it? :D